src/schemathesis/specs/openapi/examples.py

```diff
@@ -133,7 +133,7 @@ def merge_examples(
             static_parameters[parameter["type"]][parameter["name"]] = parameter["examples"][
                 min(idx, len(parameter["examples"]) - 1)
             ]
-        if "examples" in request_body_examples:
+        if "examples" in request_body_examples and len(request_body_examples["examples"]) > 0:
```

Comment:
What do you think about using `request_body_examples["examples"]` instead of  `len(request_body_examples["examples"]) > 0` which is more common for checking if a sequence is non-empty?